src/base/bittorrent/torrentcreatorthread.h

```diff
@@ -47,6 +47,8 @@ namespace BitTorrent
         void create(const QString &inputPath, const QString &savePath, const QStringList &trackers,
                     const QStringList &urlSeeds, const QString &comment, bool isPrivate, int pieceSize);
 
+        static int getPiecesCount(const QString &inputPath, const int pieceSize);
```

Comment:
Maybe function name without 'get' or with something like 'compute' prefix is better? 'Get' gives an impression of easy peeking from a shelve, but here we compute the value every time.